nameservice/tutorial/03-types.md

```diff
@@ -2,14 +2,20 @@
 order: 3
 ---
 
-
 # Types
 
-First Thing we're going to do is define a struct that holds all the metadata of a name. We will call this struct Whois after the ICANN DNS terminology.
+First thing we're going to do is create a module in the `/x/` folder with the scaffold tool using the below command:
+
+In the case of this tutorial we will be naming the module `nameservice`
+```bash
+cd x/
+
+scaffold module [user/Github org] [repo] nameservice
```

Comment:
```suggestion
scaffold module [user] [repo] nameservice
```
We should be consistent with this, github org is nice to have but i think `user` is preferable for conciseness